recipes/cmip/post-link.sh

```diff
@@ -1,5 +1,3 @@
-#!/usr/bin/env bash
```

Comment:
why is this necessary?

Reply:
Hello,

You are right, the shebang is not necessary. Please any clue on how to print a break-line instead of printing "\n" in the post-link.sh message?

Pau